File: FrontEnd/src/app/features/sales/receipts/classes/view-models/list/receipt-list-vm.ts
import { ReceiptListCustomerVM } from './receipt-list-customer-vm'
import { SimpleEntity } from '../../../../../../shared/classes/simple-entity'

export interface ReceiptListVM {

    invoiceId: string
    date: SimpleEntity
    formattedDate: string
    customer: ReceiptListCustomerVM
    documentType: SimpleEntity
    shipOwner: SimpleEntity
    paymentMethod: SimpleEntity
    invoiceNo: number
    grossAmount: number
    remarks: string
    isEmailPending: boolean
    isEmailSent: boolean
    isCancelled: boolean

}

File: API/Features/Sales/Receipts/ViewModels/List/ReceiptListVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Sales.Receipts {

    public class ReceiptListVM {

        public string InvoiceId { get; set; }
        public string Date { get; set; }
        public int InvoiceNo { get; set; }
        public SimpleEntity Customer { get; set; }
        public SimpleEntity DocumentType { get; set; }
        public SimpleEntity ShipOwner { get; set; }
        public SimpleEntity PaymentMethod { get; set; }
        public decimal GrossAmount { get; set; }
        public bool IsEmailPending { get; set; }
        public bool IsEmailSent { get; set; }
        public bool IsCancelled { get; set; }
        public string Remarks { get; set; }

    }

}